draft-scenarios/clash/gold_rush.tex

```diff
@@ -0,0 +1,163 @@
+    \end{tabularx}  
+    \medskip  
+  }  
+  {\footnotesize * Including Quick Combat. Bypassing without visiting the Field (such as with the \textit{Pathfinding} Ability) is still allowed.}  
+\end{table*}  
+
+\begin{itemize}
+  \item At the start of certain Rounds, players must send a \textbf{tranche} to their lieges. The tranche value is expressed in \svg{gold}, but may be partially or entirely paid in other resources using these exchange rates: 1 \svg{building_materials} equals 2 \svg{gold} and 1 \svg{valuables} equals 4 \svg{gold}, with no change given. In Alliance mode, allied players collectively pay a single tranche. Any player or team unable to pay loses the game immediately.  
+  \item The difficulty of all Fields on the \#C1 Tile is lowered by one Level (effectively making it a ``V-VI'' Tile).  
+  \item Certain locations have special rules, typically allowing players to acquire resource-generating Cards - refer to the table below.
+  \item A Main Hero carrying the Grail Token is not protected by the \textit{Sanctuary} effect.
+  \item The Surrendering cost for a Main Hero carrying the Grail Token is increased to 30 \svg{gold}.
+  \item When defeating an enemy Main Hero, the victorious player may, instead of taking 5 \svg{gold}, choose to take either the Grail Token or one Artifact from the opponent's Discard Pile.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}  
+
+\begin{itemize}  
+  \item At the end of any Round, a player/team who possesses the Grail Token and can afford all remaining \textbf{tranches} wins immediately.  
+  \item In \textbf{PvP games}, if no player wins after Round 13, the winner is the player/team who is able to pay the highest \textbf{tranche} value. When calculating this final value, possession of the Grail Token is worth an additional 60 \svg{gold}.  
+\end{itemize}  
+
+\subsection*{\MakeUppercase{Defeat Conditions}}  
+
+\begin{itemize}  
+  \item Any player/team unable to pay a required \textbf{tranche} loses the game immediately.  
+  \item In \textbf{Solo play}, the player loses if they fail to meet the Victory Condition by the end of Round 13.  
+  \item In \textbf{PvP games}, if all players/teams cannot pay their \textbf{tranche} in the same Round, the player/team closest to affording their \textbf{tranche} wins (with the Grail Token counting as 60 \svg{gold}).  
+\end{itemize}  
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+Players must pay tranches at the start of specific Rounds, as shown in the table below. Tranches due on Resource Rounds are paid \textbf{after} receiving resources and resolving all other Round-beginning effects.  
+
+After paying a tranche, \textbf{remove all Black Cubes} from every Windmill and Water Wheel on the map.  
+
+\vfill
+
+\hommtable[]{16}{
+  \centering
+  \medskip
+  \textbf{Tranches depending on the variant}\\
+  \bigskip
+  \begin{tabularx}{\linewidth}{XXXXX} \darkcell{Round} & \darkcell{Solo} & \darkcell{1v1} & \darkcell{2v2} \\
```

Comment:
```suggestion
  \begin{tabularx}{0.95\linewidth}{XXXXX} \darkcell{Round} & \darkcell{Solo} & \darkcell{1v1} & \darkcell{2v2} \\
```
This will make the table symmetric.